checklists/alz_checklist.en.json

```
@@ -2534,7 +2538,8 @@
             "guid": "0cadb8c7-8fa5-4fbf-8f39-d1fadb3b0460",
             "id": "H01.05",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/considerations/development-strategy-development-lifecycle#automated-builds"
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/considerations/development-strategy-development-lifecycle#automated-builds",
+            "training": "https://learn.microsoft.com/en-us/training/modules/work-linting-unit-test-github-actions/"
```

This module is specific to ML applications which may not be as relatable to Azure customers that are building other types of workloads.  @alisakina99 - I will humbly suggest the following if you agree:

```suggestion
            "training": "https://learn.microsoft.com/training/modules/run-quality-tests-build-pipeline/"
```